File: java/src/main/java/co/wangming/crud/crudrestful/mapper/HelloWorldMapper.java
package co.wangming.crud.crudrestful.mapper;

import co.wangming.crud.crudrestful.model.HelloWorld;
import org.apache.ibatis.annotations.Mapper;
import org.apache.ibatis.annotations.Select;

import java.util.List;

/**
 * Created By WangMing On 2020-01-20
 **/
@Mapper
public interface HelloWorldMapper {

    @Select("select * from helloworld")
    List<HelloWorld> selectAll();

}

File: kotlin/src/main/kotlin/co/wangming/crud/crudrestful/mapper/HelloWorldMapper.kt
package co.wangming.crud.crudrestful.mapper

import co.wangming.crud.crudrestful.model.HelloWorld
import org.apache.ibatis.annotations.Mapper
import org.apache.ibatis.annotations.Select

/**
 *
 * Created By WangMing On 2019-12-10
 **/
@Mapper
interface HelloWorldMapper {

    @Select("select * from helloworld")
    fun selectAll(): List<HelloWorld>?
}